demo/demo_primary.py

```diff
@@ -688,6 +688,10 @@ def get_time_attestation_for_ecu(ecu_serial):
     #     'to CAN IDs. Sending time attestation back.')
 
     print('Distributing time attestation to ECU ' + repr(ecu_serial))
+    # If we're using ASN.1/DER, then the attestation is binary data we're about
+    # to transmit via XMLRPC, so we should wrap it appropriately:
+    attestation = xmlrpc_client.Binary(
+        primary_ecu.get_last_timeserver_attestation())
```

Comment:
The comment and code don't seem to match.

The code always seems to send the `attestation` in Binary, but the comment says that "If we're using ASN.1/DER, the attestation is wrapped in binary data..."

I was expecting the attestation to be wrapped in Binary if tuf.conf.METADATA_FORMAT == 'der'.